packages/react-core/src/components/SearchInput/__tests__/SearchInput.test.tsx

```diff
@@ -102,7 +103,42 @@ describe('SearchInput', () => {
 
     userEvent.click(screen.getByRole('button', { name: 'Search' }));
 
+    userEvent.click(screen.getByRole('button', { name: 'Open advanced search' }));
+    expect(screen.getByTestId('test-id').querySelector('.pf-c-panel')).toBeInTheDocument()
```

Comment:
Nit: rather than using `querySelector` + `toBeInTheDocument` to determine where the search menu is being placed in the DOM can you use an RTL query to select the form (or something in it) and maybe the `toContainElement` matcher?

Reason being that I believe this would better replicate the actual user experience with the component.

Reply:
Good idea. Thanks 👍🏼